src/AspNet.Security.OAuth.LinkedIn/LinkedInAuthenticationOptions.cs

```diff
@@ -25,19 +26,18 @@ public LinkedInAuthenticationOptions()
         UserInformationEndpoint = LinkedInAuthenticationDefaults.UserInformationEndpoint;
         EmailAddressEndpoint = LinkedInAuthenticationDefaults.EmailAddressEndpoint;
 
-        Scope.Add("r_liteprofile");
-        Scope.Add("r_emailaddress");
+        Scope.Add("openid");
+        Scope.Add("profile");
+        Scope.Add("email");
 
         ClaimActions.MapJsonKey(ClaimTypes.NameIdentifier, ProfileFields.Id);
-        ClaimActions.MapCustomJson(ClaimTypes.Name, user => GetFullName(user));
-        ClaimActions.MapCustomJson(ClaimTypes.GivenName, user => GetMultiLocaleString(user, ProfileFields.FirstName));
-        ClaimActions.MapCustomJson(ClaimTypes.Surname, user => GetMultiLocaleString(user, ProfileFields.LastName));
-        ClaimActions.MapCustomJson(Claims.PictureUrl, user => GetPictureUrls(user)?.LastOrDefault());
-        ClaimActions.MapCustomJson(Claims.PictureUrls, user =>
-        {
-            var urls = GetPictureUrls(user);
-            return urls == null ? null : string.Join(',', urls);
-        });
+        ClaimActions.MapJsonKey(ClaimTypes.Name, ProfileFields.Name);
+        ClaimActions.MapJsonKey(ClaimTypes.Email, ProfileFields.Email);
+        ClaimActions.MapJsonKey(ClaimTypes.GivenName, ProfileFields.GivenName);
+        ClaimActions.MapJsonKey(ClaimTypes.Surname, ProfileFields.FamilyName);
```

Comment:
The comments on various new constants you've added are noted as being multilocale strings, but then you're not using the code to handle them (in fact, you've completely deleted it).

Either the comments are wrong, or you're not handling the values correctly.